doc/model.rst

```diff
@@ -0,0 +1,739 @@
+*****
+Model
+*****
+========
+Overview
+========
+SmartSim ``Model`` objects enable users to execute computational tasks in an
+``Experiment`` workflow, such as launching compiled applications,
+running scripts, or performing general computational operations. ``Models`` can be launched with
+other SmartSim entities and infrastructure to build AI-enabled workflows.
```

Comment:
What do we mean by "infrastructure" here?

Reply:
I believe ML models and TorchScripts